internal/service/service.go

```diff
@@ -61,7 +65,33 @@ func (s *Service) Run() error {
 	if s.Port == 0 {
 		s.Port = port
 	}
-	return http.ListenAndServeTLS(fmt.Sprintf(":%d", s.Port), s.CertFile, s.KeyFile, s.Routes())
+
+	cert, err := tls.LoadX509KeyPair(s.CertFile, s.KeyFile)
+	if err != nil {
+		return fmt.Errorf("tls.LoadX509KeyPair(%s, %s) failed: ", s.CertFile, s.KeyFile)
+	}
+
+	addr := fmt.Sprintf(":%d", s.Port)
+	config := &tls.Config{
+		Certificates: []tls.Certificate{cert},
+		MinVersion:   tls.VersionTLS12,
+	}
+
+	server := &http.Server{
+		Addr:              addr,
+		ReadHeaderTimeout: 5 * time.Second,
```

Comment:
1. Is 'ReadHeaderTimeout' only for TLS listener, or for all Read operation?
2. Is 5sec a recommended timeout value?
3. Better to use a constant variable for the time out setting.

Reply:
1. `ReadHeaderTimeout` is for the new `server` instance, and the instance is for TLS listener. It could say 'ReadHeaderTimeout' only for TLS listener
2. There is no recommended value. But [PowerStore ]('ReadHeaderTimeout' only for TLS listener) uses 5 the same.
3. I flip through all the source code including CSM, 3rd, GO package, timeout values are all set as plain value, not constant. I think it should be good for us as well.